dags/bqetl_kpi_dashboard.py

```diff
@@ -0,0 +1,46 @@
+# Generated via query_scheduling/generate_airflow_dags 
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {'owner': 'jklukas@mozilla.com', 'email': ['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com'], 'depends_on_past': False, 'start_date': datetime.datetime(2020, 5, 12, 0, 0), 'retry_delay': 'datetime.timedelta(seconds=600)', 'email_on_failure': True, 'email_on_retry': True, 'retries': 1}
```

Comment:
Would it be fairly easy to run `black` on the Jinja output before writing out the file? It would be nice to have this dictionary split reasonably to multiple lines, but seems silly to add logic in the jinja template to make that happen.

Reply:
Yeah, I can just add it as another step to `script/generate_airflow_dags` after the DAGs have been generated.